src/ws/test-remotectlcertificate.c

```diff
@@ -251,6 +295,8 @@ main (int argc,
 {
   cockpit_test_init (&argc, &argv);
 
+  openssl_path = g_find_program_in_path ("openssl");
```

Comment:
Neat idea to save the string.  I almost suggested doing that, except that we don't actually use the value.

In any case, maybe you want to free it before we exit?  It shouldn't trigger any problems with valgrind (since it's still very much reachable), so I leave the choice to you.

Reply:
Right, I'd only be concerned about valgrind. I don't see much point in doing extra work that the kernel does anyway, honestly.